app/Http/Controllers/TaskController.php

```diff
@@ -124,9 +125,20 @@ public function show(Request $request, Task $task)
             return to_route('task.index')->withErrors(['access_error' => 'アクセスが無効です。']);
         }
 
-        return view('task.show', [
-            'task' => $task
-        ]);
+        $task_comments = TaskComment::where('task_id', $task->id)
+            ->orderBy('created_at', 'asc')
+            ->get();
+
+        if(is_null($task_comments)){
+            return view('task.show', [
+                'task' => $task
+            ]);
+        }else{
+            return view('task.show', [
+                'task' => $task,
+                'comments' => $task_comments
+            ]);
+        }
```

Comment:
null判定はbladeでしてください。
```suggestion
        return view('task.show', [
                'task' => $task,
                'comments' => $task_comments
            ]);
```